base_layer/core/src/transaction_protocol/sender.rs

```diff
@@ -262,7 +268,7 @@ impl SenderTransactionProtocol {
             .with_signature(&s_agg)
             .build()?;
         tx_builder.with_kernel(kernel);
-        let tx = tx_builder.build()?;
+        let tx = tx_builder.build(prover, factory)?;
```

Comment:
nit: ln 271 and 272 can be combined  as only `tx_builder.build(prover, factory)`

Reply:
Good catch, thanks

Reply:
Fixed.